### ISSUE TITLE
Regarding the issue of training loss

### ISSUE THREAD

**Issue by ccqq7** [yanghangAI/DKiS#4]
When I tried to train again, I found that the loss value was very high, reaching 50,000. Could you please tell me if this phenomenon is normal?

**Comment by yanghangAI** [comment]
If the loss value stay 50,000 for many epochs, then it is abnormal. Did you standardize the image during the training?
